### PR TITLE
Qt: Update Texture Filtering Description

### PR DESCRIPTION
**Pull request by seta-san**
### Description of Changes
adds a description of texture filtering

### Rationale behind Changes
old description wasn't a real description

### Suggested Testing Steps
if it compiles and text shows it's fine.

### CODE REVIEW

**Review by lightningterror — COMMENTED** [review]
It was fine as it was, I wouldn't change that.

**Comment by TheTechnician27** [comment]
Imo, the current tooltip is literally useless. "Texture filtering controls the texture filtering of the emulation" is about on par with "The floor is made of floor".

**Comment by lightningterror** [comment]
Maybe, but at the very least the tooltip can be shortened.

**Review by F0bes — CHANGES_REQUESTED** [review]
If you really want to change this; I would prefer the structured layout dithering has, rather than a paragraph.
Also, your PS2 description is too wordy.
We should probably change the nearest option to `Nearest (Forced)` as well to follow the other settings.

**Review by JordanTheToaster — APPROVED** [review]

**Review by lightningterror — APPROVED** [review]
Looks much better now.

**Review by F0bes — APPROVED** [review]
